src/main/java/wooteco/subway/web/ControllerAdvice.java

```diff
@@ -0,0 +1,39 @@
+package wooteco.subway.web;
+
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.MethodArgumentNotValidException;
+import org.springframework.web.bind.annotation.ExceptionHandler;
+import org.springframework.web.bind.annotation.RestControllerAdvice;
+
+import sun.dc.path.PathException;
```

Comment:
```suggestion
import wooteco.subway.web.exception.path.PathException;
```
다른게 들어갔네요. 실제로도 아차 싶으면 이럴 때가 있어서 한번 더 신경써주시는게 좋습니다. 😉